### ISSUE TITLE
[BUG] Unable to add trakt account on xbox

### ISSUE THREAD

**Issue by 101ben1011** [nixgates/plugin.video.seren#312]
Firstly thanks for all your hard work on Seren!

I run kodi 18.9 on the xbox series x.

I went straight from v1 to v2.08.

Everything seemed to go fine, but when I opened my trakt tv collection it was a blank page. However movies worked fine.

Tried clearing all caches but was no better, so I uninstalled seren and reinstalled it. Now I am unable to authorise trakt. It goes through the process to pair them, then I get a successful message from seren, immediately followed by a seren error.

Seren tries to sync with trakt when I open kodi and now and then while kodi is running, even though in the accounts page, there is no trakt account active.

I have attached a photo of the error log. Sorry for the poor pic, dont know how else to get the error log off the xbox lol.

![20210103_182224](https://user-images.githubusercontent.com/76916513/103485941-feaade00-4df1-11eb-9ad2-0e6205358185.jpg)

Update: so seren has just ran a sync which seemed to work. I am now back to where I started. It is showing my my film collection. The TV collection is empty. The accounts settings page does not show a trakt account as being active.
It seems to be half authorised somehow.

Update 2- seems to be the same for my real debrid account. Said it was successful, but not showing in accounts and if I try to watch anything, I get a seren message to say I need a debrid account.

**Comment by nixgates** [comment]
@101ben1011 can I please get a log of you trying to auth Trakt and your debrid account. Thank you :+1:

**Comment by fairlynew** [comment]
Would like to add.
XBOne. Clean install of kodi & Seren.
Trakt not fully syncing.
Oddly, it shows a personal trakt list I created,
But not next up, upcoming, watchlist, new tv shows, collection.

Same as OP, it runs a sync, seems to finish, but on a restart picks back up at sync watched shows, sync collected shows, and switches back and fortha few seconds.  

Not expecting an answer, just trying to get you info.

**Comment by 101ben1011** [comment]
Hi,
I dont fully understand the logs lol and can't find a way to get the actual log file off the xbox.
I had hoped the pic I attached would help which was the error created just after I tried to add my trakt account( it means nothing to me!).
I take it you need more info? Or do you need to see more of the log file? I would probably have to take multiple pictures with it scrolled up if thats any use?
Thanks.

**Comment by nixgates** [comment]
You can more easily log an issue using the log uploader addon from the kodi repository! Makes it super simple, just try to auth, run the log uploader addon and it will give you a link for me to look at.

Unfortunately that picture you've attached while is helpful in some ways for the sync issue, doesn't tell me much about the RD issue unfortunately.

**Comment by 101ben1011** [comment]
Ah ok. Thanks. The log viewer doesn't work on xbox but the uploader does. I'll do that later when I'm back at home.

**Comment by 101ben1011** [comment]
Hi. I think I have done it properly this time lol.

Link to log file:
https://paste.kodi.tv/otadisaweh

Actions I took prior to uploading the log:
1. Opened kodi
2. Updated seren to 2.0.9
3. Cleared seren app data
4. Tried to authorise my real debrid, which said successful, followed by a seren error as before.
5. Tried to authorise my trakt account, which said successful, followed by a seren error.

My accounts page is still blank with no accounts authorised.

The only thing I haven't tried yet is a complete kodi reinstall, which I could try if you think it might be a dependency issue or something, which isn't being cleared by reinstalling seren.

Hope this helps, if I can help in any other way, let me know, and thanks again for your efforts in trying to get it sorted.
